Shell/Statistics.hpp

```diff
@@ -188,6 +200,11 @@ class Statistics
   unsigned taNegativeInjectivitySimplifications;
   unsigned taAcyclicityGeneratedDisequalities;
 
+  //to be moved to the property object once that 
+  //is controlled by environment
+  bool higherOrder;
+  bool polymorphic;
```

Comment:
Didn't we resolve the problem with Property already?

Reply:
No, not yet.